### PR TITLE
bpo-29247: Document return value of epoll.poll()

### PR DESCRIPTION
**Pull request by berkerpeksag**
https://bugs.python.org/issue29247

### CODE REVIEW

File: Doc/library/select.rst [hunk 1]
```diff
@@ -327,30 +296,65 @@ Edge and Level Trigger Polling (epoll) Objects
...
 
-   .. versionchanged:: 3.5
-      The function is now retried with a recomputed timeout when interrupted by
-      a signal, except if the signal handler raises an exception (see
-      :pep:`475` for the rationale), instead of raising
-      :exc:`InterruptedError`.
+
+The following constants can be used with
+:meth:`~epoll.register` and :meth:`~epoll.modify` methods.  See
+:manpage:`epoll(4)` for more information.
```
Comment (csabella) [inline Doc/library/select.rst:319]: I wonder if this table should be under `epoll.register`, similar to the section for `poll`.

File: Doc/library/select.rst [hunk 2]
```diff
@@ -264,48 +264,17 @@ object.
...
-   +-------------------------+-----------------------------------------------+
-   | :const:`EPOLLRDBAND`    | Priority data band can be read.               |
-   +-------------------------+-----------------------------------------------+
-   | :const:`EPOLLWRNORM`    | Equivalent to :const:`EPOLLOUT`               |
-   +-------------------------+-----------------------------------------------+
-   | :const:`EPOLLWRBAND`    | Priority data may be written.                 |
-   +-------------------------+-----------------------------------------------+
-   | :const:`EPOLLMSG`       | Ignored.                                      |
-   +-------------------------+-----------------------------------------------+
+.. method:: epoll.poll(timeout=-1, maxevents=-1)
```
Comment (csabella) [inline Doc/library/select.rst:267]: Was there a reason for moving epoll.poll?  The other sections (devpoll and poll) both have the `.poll` method defined last in the section.  I don't know which is better, but moving this makes it inconsistent.

File: Doc/library/select.rst [hunk 3]
```diff
@@ -264,48 +264,17 @@ object.
...
+
+   Wait for events. *timeout* in seconds (float)  Return a list of
```
Comment (csabella) [inline Doc/library/select.rst:269]: Can `timeout` be explained better?  It's not really a proper sentence here.  Also, `maxevents` isn't explained.  Is it the number of tuples in the list?

File: Doc/library/select.rst [hunk 4]
```diff
@@ -314,7 +283,7 @@ Edge and Level Trigger Polling (epoll) Objects
 
 .. attribute:: epoll.closed
 
-   ``True`` if the epoll object is closed.
+   Return ``True`` if the epoll object is closed.
```
Comment (csabella) [inline Doc/library/select.rst:286]: Should `Return` be used for an attribute?  Maybe use `Set to`?

**Comment by vstinner** [comment]
I removed the " needs backport to 3.6" label, the 3.6 branch no longer accept bugfixes (only security fixes are accepted): https://devguide.python.org/#status-of-python-branches

**Comment by JulienPalard** [comment]
@berkerpeksag, ping

**Comment by JelleZijlstra** [comment]
Closing as it's been several years and the review comments weren't addressed. Also, there are several conflicts.

**Pull request closed without merging**